sql/updates/post-update.sql

```diff
@@ -30,3 +30,46 @@ $$;
 
 -- can only be dropped after views have been rebuilt
 DROP FUNCTION IF EXISTS _timescaledb_internal.cagg_watermark(oid);
+
+-- For objects that are newly created, we need to set the initprivs to
+-- the initprivs for some table that was created in the installation
+-- of the TimescaleDB extension and not as part of any update.
+--
+-- We chose the "chunk" catalog table for this since that is created
+-- in the first version of TimescaleDB and should have the correct
+-- initprivs, but we could use any other table that existed in the
+-- first installation.
+INSERT INTO saved_privs (
+     SELECT nspname, relname, relacl,
+ 	    (SELECT tmpini FROM saved_privs
+	      WHERE tmpnsp = '_timescaledb_catalog' AND tmpname = 'chunk')
+       FROM pg_class JOIN pg_namespace ns ON ns.oid = relnamespace
+		LEFT JOIN saved_privs ON tmpnsp = nspname AND tmpname = relname
+      WHERE nspname IN ('_timescaledb_catalog', '_timescaledb_config')
+ 	 OR nspname = '_timescaledb_internal'
+        AND relname IN ('hypertable_chunk_local_size', 'compressed_chunk_stats',
+                        'bgw_job_stat', 'bgw_policy_chunk_stats')
+) ON CONFLICT DO NOTHING;
```

Comment:
```suggestion
INSERT INTO saved_privs
     SELECT nspname, relname, relacl,
 	    (SELECT tmpini FROM saved_privs
	      WHERE tmpnsp = '_timescaledb_catalog' AND tmpname = 'chunk')
       FROM pg_class JOIN pg_namespace ns ON ns.oid = relnamespace
		LEFT JOIN saved_privs ON tmpnsp = nspname AND tmpname = relname
      WHERE nspname IN ('_timescaledb_catalog', '_timescaledb_config')
 	 OR nspname = '_timescaledb_internal'
        AND relname IN ('hypertable_chunk_local_size', 'compressed_chunk_stats',
                        'bgw_job_stat', 'bgw_policy_chunk_stats')
ON CONFLICT DO NOTHING;
```
those parens are not needed